yt_dlp/extractor/yle_areena.py

```diff
@@ -52,20 +94,33 @@ def _real_extract(self, url):
                     'name': sub.get('kind'),
                 })
 
+        # Video manifest URL
+        manifest_url = traverse_obj(video_data, ('data', 'ongoing_ondemand', 'manifest_url'))
+        # Extract video release date
+        start_time = traverse_obj(video_data, ('data', 'ongoing_ondemand', 'start_time'), expected_type=str)
+        release_date = '%s%s%s' % (start_time[0:4], start_time[5:7], start_time[8:10])
+        # Service URL
+        kaltura_url = 'kaltura:1955031:%s' % traverse_obj(video_data, ('data', 'ongoing_ondemand', 'kaltura', 'id'))
+        # Include video source URL for Kaltura internal use
+        smuggle_data = smuggle_url(kaltura_url, {'source_url': url})
+
         return {
             '_type': 'url_transparent',
-            'url': 'kaltura:1955031:%s' % traverse_obj(video_data, ('data', 'ongoing_ondemand', 'kaltura', 'id')),
+            'url': smuggle_data,
             'ie_key': KalturaIE.ie_key(),
             'title': (traverse_obj(video_data, ('data', 'ongoing_ondemand', 'title', 'fin'), expected_type=str)
                       or episode or info.get('title')),
             'description': description,
             'series': (traverse_obj(video_data, ('data', 'ongoing_ondemand', 'series', 'title', 'fin'), expected_type=str)
                        or series),
             'season_number': (int_or_none(self._search_regex(r'Kausi (\d+)', description, 'season number', default=None))
-                              or int(season_number)),
+                              or int_or_none(season_number)),
             'episode_number': (traverse_obj(video_data, ('data', 'ongoing_ondemand', 'episode_number'), expected_type=int_or_none)
-                               or int(episode_number)),
+                               or int_or_none(episode_number)),
             'thumbnails': traverse_obj(info, ('thumbnails', ..., {'url': 'url'})),
             'age_limit': traverse_obj(video_data, ('data', 'ongoing_ondemand', 'content_rating', 'age_restriction'), expected_type=int_or_none),
             'subtitles': subtitles,
+            'manifest_url': manifest_url,
+            'webpage_url': url,
```

Comment:
webpage_url is auto-generated. manifest_url makes no sense for `_type=url_transparent`
```suggestion
```